content/docs/guides/contribute/contribute-code/commit-conventions.en.md

```diff
@@ -46,4 +48,32 @@ Welcome to ease review, but do not merge:
 - `fixup! previous commit`: an [autosquash](../share-changes) must be run before the merge
 - `Revert "previous commit of the same PR"`: both commits must be dropped before merging
 
+## The Developer Certificate of Origin (DCO)
+
+All of OSRD's projects use the DCO (Developer Certificate of Origin) to address
+legal matters. The DCO helps confirm that you have the rights to the code you
+contribute. For more on the history and purpose of the DCO, you can read [The
+Developer Certificate of Origin](https://bssw.io/blog_posts/the-developer-certificate-of-origin)
+by Roscoe A. Bartlett.
+
+To comply with the DCO, **all commits must include a Signed-off-by line**.
+
+### How to sign a commit using git in a shell ?
+
+To sign off a commit, simply add the `-s` flags to your `git commit` command,
+like so:
+
+```bash
+git commit -s -m "Your commit message"
+```
+This also applies when using the `git revert` command.
+
+### How to do sign a commit using git in Visual Studio Code (VS Code) ?
+
+Now, go in `Files` -> `Preferences` -> `Settings` and search for **Always Sign
+Off**. (This only has to be done once per repository.)
```

Comment:
This sentence seems strange to me. What did you mean by that?

Note that you have two kind of settings in vs code. One is for the **user** (global to vs code). The other one is local to the **workspace** (which is not the same as the repository)

Reply:
I meant the workspace but after checking again it appears that this setting is bind to the user. Thanks for catching that mistake.

Reply:
I checked and this setting (like all of them) can be bind to the workspace.

![image](https://github.com/user-attachments/assets/99aa8173-517a-444a-9c33-1f4d119675d3)